README.md

```diff
@@ -57,17 +64,25 @@ You can find a helm chart [here](https://artifacthub.io/packages/helm/pleco/plec
 ---
 ## Requirements
 
-In order to make pleco check and clean expired aws resources you need to set the following environment variables:
+In order to make pleco check and clean expired resources you need to set the following environment variables:
+#### For AWS
 ```bash
 $ export AWS_ACCESS_KEY_ID=<access_key>
 $ export AWS_SECRET_ACCESS_KEY=<secret_key>
 ```
+
+#### For Scaleway
+```bash
+$ export SCALEWAY_ACCESS_KEY=<access_key>
+$ export SCALEWAY_SECRET_KEY=<secret_key>
+$ export SCALEWAY_ORGANISATION_ID=<organization_id>
```

Comment:
Shouldn't it be `SCALEWAY_PROJECT_ID`? Does organization represents Qovery's org? If not, I guess it should be Scaleway project.